developer/src/kmc-package/src/compiler/package-validation.ts

```diff
@@ -40,21 +52,37 @@ export class PackageValidation {
     return true;
   }
 
-  private checkLexicalModels(kmpJson: KmpJsonFile.KmpJsonFile): boolean {
-    if(kmpJson.lexicalModels) {
-      for(let model of kmpJson.lexicalModels) {
-        this.checkForDuplicatedLanguages('model', model.id, model.languages);
-      }
+  private checkLexicalModels(filename: string, kmpJson: KmpJsonFile.KmpJsonFile): boolean {
+    if(!kmpJson.lexicalModels || kmpJson.lexicalModels.length == 0) {
+      return true;
```

Comment:
When would checkLexicalModels or checkKeyboards ever return false?

Would it be
```
if (kmpJson.lexicalModels && kmpJson.lexicalModels.length == 0) {
  return false;
```

Reply:
Was thinking this myself until I remembered that they'll return false once there are errors generated rather than warnings.